packages/domains-table/src/domains-table/__tests__/domains-table-row.tsx

```diff
@@ -437,12 +437,14 @@ describe( 'expires or renew on cell', () => {
 			has_registration: false,
 		} );
 
-		render( <DomainsTableRow domain={ partialDomain } />, {
+		const { container } = render( <DomainsTableRow domain={ partialDomain } />, {
 			domains: [ partialDomain ],
 			isAllSitesView: true,
 		} );
 
-		expect( screen.getByText( '-' ) ).toBeInTheDocument();
+		const renewNowCells = container.querySelectorAll( '.domains-table-row__renews-on-cell' );
```

Comment:
I'd prefer not to rely on a class name here, but the cell doesn't have much else to target (no role or unique content)

Reply:
cc @nightnei

Reply:
Excellent - I've gotten so used (in the past) to avoiding `data-testid`s that they didn't pop into my head until you suggested it. This test has been updated with a `data-testid` so it will continue to pass after the feature flag is removed. As we discussed earlier, we can look into alternate methods in a follow up if needed.

I haven't added new tests yet, and will need to take another look in the morning. There's a bit of a wrinkle, because even if we update the default ssl value as suggested, and/or pass a custom value, the `sslStatus` isn't parsed the same way during these tests as it would be in production. Because we're using test data and not an actual domain, [`useSiteDomainsQuery` comes back `undefined`](https://github.com/Automattic/wp-calypso/blob/01858f4725f9c3eb1050bd9df4b99fa999efe3e5/packages/domains-table/src/use-domain-row.ts#L30-L38). This, in turn, means `sslStatus` is `undefined`, making testing less than ideal. I'll investigate more in tomorrow.